app/views/about/index.html.erb

```diff
@@ -20,9 +20,28 @@
           For this app we built our own open source Microservice to provides the API endpoints that are needed. Using an impressive dataset provided by Jet Propulsion Laboratory, combined with a Skyfield - Python package that performs complex calculations - we were able to build out a stand alone Python application which delivers the information that we consume through the Space In Your Face Rails application.
         </p>
       </section>
+      <h1>How to View Planets</h1>
+      <section class='about_box' id='telescope_instructions'>
+        <p>In order to look at planets with the naked eye, you just need to turn the right direction
```

Comment:
We might add in the future a suggestion to use a compass or compass app on the user's phone.